### PR TITLE
refactor: onboarding style changes

### PR DESCRIPTION
**Pull request by TGlide**
## What does this PR do?

- Changes some styles in the onboarding to better fit design schemas
- Changes copy in onboarding and Add Platform wizards

## Test Plan

Manual

## Related PRs and Issues

N/A

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

**Comment by gewenyu99** [comment]
@TGlide Can you tag me on what I need to review?

Also, let's make sure we hit all the copywriting changes for the onboarding changes. I'll setup a quick sync with you about this.

**Comment by TGlide** [comment]
@gewenyu99 all of the files are mostly copy review regarding new copy on Onboarding and Add Platform pages. But you can also go through the preview link on Vercel to check it out if you prefer!

File: src/routes/console/project-[project]/overview/platforms/wizard/android/step2.svelte [hunk 1]
```diff
@@ -11,8 +11,8 @@
 
 <WizardStep>
     <svelte:fragment slot="title">Get the SDK</svelte:fragment>
-    <p>First, add this to your root level build.gradle file:</p>
+    <p>First, add this to your root level <code class="inline-code">build.gradle</code> file.</p>
```
Comment (gewenyu99) [inline src/routes/console/project-[project]/overview/platforms/wizard/android/step2.svelte:14]: @TGlide Can you find someone who's familiar with the Android ecosystem to see if the two sentences make sense?

All this talk about different grade files is getting a little confusing.

File: src/routes/console/project-[project]/overview/platforms/wizard/flutter/step1.svelte [hunk 2]
```diff
@@ -106,10 +106,19 @@
...
+        [Platform.Linux]: 'package name',
+        [Platform.Macos]: 'bundle ID',
+        [Platform.Windows]: 'package name',
+        [Platform.Web]: 'hostname'
+    }[platform];
 </script>
 
 <WizardStep {beforeSubmit}>
-    <svelte:fragment slot="title">Register your Flutter app</svelte:fragment>
+    <svelte:fragment slot="title">Register your {registee}</svelte:fragment>
```
Comment (gewenyu99) [inline src/routes/console/project-[project]/overview/platforms/wizard/flutter/step1.svelte:121]: AMAZING!!!!

**Review by gewenyu99 — CHANGES_REQUESTED** [review]
Excited to see it through :DDD

File: src/routes/console/project-[project]/overview/platforms/wizard/android/step2.svelte (hunk 1 above)
Comment (TGlide) [reply src/routes/console/project-[project]/overview/platforms/wizard/android/step2.svelte:14]: @lohanidamodar maybe?

Comment (gewenyu99) [reply src/routes/console/project-[project]/overview/platforms/wizard/android/step2.svelte:14]: or @abnegate Heehee

**Review by ArmanNik — APPROVED** [review]

File: src/routes/console/project-[project]/overview/platforms/wizard/web/step3.svelte [hunk 3]
```diff
@@ -12,12 +13,12 @@ client
...
-    <svelte:fragment slot="title">Let's get coding</svelte:fragment>
+    <svelte:fragment slot="title">Initialize SDK</svelte:fragment>
 
-    <h2 class="heading-level-7">Init your SDK</h2>
+    <h2 class="heading-level-7">Initialize your SDK</h2>
     <p>
-        Now that you've downloaded the SDK, it's time to initialze it. Use your project ID, which
-        can be found in your project settings page.
+        Initialize your SDK by pointing the client to your Appwrite project using your <Id
+            value={project}>Project ID</Id>
```
Comment (gewenyu99) [inline src/routes/console/project-[project]/overview/platforms/wizard/web/step3.svelte:21]:
```suggestion
            value={project}>Project ID</Id>.
```

Missing period.

File: src/routes/console/project-[project]/overview/platforms/wizard/flutter/step3.svelte [hunk 4]
```diff
@@ -19,10 +20,10 @@ client
 <WizardStep>
     <svelte:fragment slot="title">Let's get coding</svelte:fragment>
 
-    <h2 class="heading-level-7">Init your SDK</h2>
+    <h2 class="heading-level-7">Initialize your SDK</h2>
     <p>
-        Now that you've downloaded the SDK, it's time to initialze it. Use your project ID, which
-        can be found in your project settings page.
+        Initialize your SDK by pointing the client to your Appwrite project using your <Id
```
Comment (gewenyu99) [inline src/routes/console/project-[project]/overview/platforms/wizard/flutter/step3.svelte:25]: This is copyable right? :P

File: src/routes/console/project-[project]/overview/platforms/wizard/flutter/step3.svelte [hunk 5]
```diff
@@ -19,10 +20,10 @@ client
 <WizardStep>
     <svelte:fragment slot="title">Let's get coding</svelte:fragment>
```
Comment (gewenyu99) [inline src/routes/console/project-[project]/overview/platforms/wizard/flutter/step3.svelte:21]:
```suggestion
    <svelte:fragment slot="title">Initialize SDK</svelte:fragment>
```

Looks like you missed one :P

File: src/routes/console/project-[project]/overview/platforms/wizard/apple/step3.svelte [hunk 6]
```diff
@@ -18,22 +19,19 @@ let client = Client()
 <WizardStep>
     <svelte:fragment slot="title">Let's get coding</svelte:fragment>
```
Comment (gewenyu99) [inline src/routes/console/project-[project]/overview/platforms/wizard/apple/step3.svelte:20]:
```suggestion
    <svelte:fragment slot="title">Initialize SDK</svelte:fragment>
```

File: src/routes/console/project-[project]/overview/platforms/wizard/flutter/step3.svelte [hunk 7]
```diff
@@ -19,10 +20,10 @@ client
...
 
-    <h2 class="heading-level-7">Init your SDK</h2>
+    <h2 class="heading-level-7">Initialize your SDK</h2>
     <p>
-        Now that you've downloaded the SDK, it's time to initialze it. Use your project ID, which
-        can be found in your project settings page.
+        Initialize your SDK by pointing the client to your Appwrite project using your <Id
+            value={project}>Project ID</Id>
```
Comment (gewenyu99) [inline src/routes/console/project-[project]/overview/platforms/wizard/flutter/step3.svelte:26]:
```suggestion
            value={project}>Project ID</Id>.
```

File: src/routes/console/project-[project]/overview/platforms/wizard/web/step3.svelte (hunk 3 above)
Comment (gewenyu99) [inline src/routes/console/project-[project]/overview/platforms/wizard/web/step3.svelte:21]:
```suggestion
            value={project}>Project ID</Id>.
```

File: src/routes/console/project-[project]/overview/platforms/wizard/android/step3.svelte [hunk 8]
```diff
@@ -16,10 +17,10 @@ val client = Client(context)
 <WizardStep>
     <svelte:fragment slot="title">Let's get coding</svelte:fragment>
```
Comment (gewenyu99) [inline src/routes/console/project-[project]/overview/platforms/wizard/android/step3.svelte:18]:
```suggestion
    <svelte:fragment slot="title">Initialize</svelte:fragment>
```

**Review by gewenyu99 — CHANGES_REQUESTED** [review]

File: src/routes/console/project-[project]/overview/platforms/wizard/flutter/step3.svelte (hunk 4 above)
Comment (TGlide) [reply src/routes/console/project-[project]/overview/platforms/wizard/flutter/step3.svelte:25]: Yes!

**Review by gewenyu99 — APPROVED** [review]
Check to make sure you're not missing periods and merge XD

**Pull request merged**